apps/frontend/src/components/xpress/XpressProposalTable.tsx

```diff
@@ -374,7 +399,47 @@ const XpressProposalTable = ({ confirm }: { confirm: WithConfirmType }) => {
 
   const statusColumn = () => [
     {
-      title: 'Status',
+      title: (
+        <>
+          <span>
+            Status
+            <Tooltip
+              title={
+                <span>
+                  <p>Tips: </p>
+                  <p>
+                    1.Change the status of a proposal to Under Review to enable
+                    experimental area selection.
+                  </p>
+                  <p>
+                    2.Status can be changed to Approved / Unsuccessful, once
+                    experimental area selection and review is completed.
```

Comment:
Maybe something like this makes it clearer that an experimental area doesn't need to be selected for you to change to Unsuccessful?

>Status can be changed to Unsuccessful, or Approved after an experimental area is selected.